components/HistoryTable.qml

```diff
@@ -52,6 +54,22 @@ ListView {
             + translationManager.emptyString;
     }
 
+    function lookupPaymentID(paymentId) {
```

Comment:
Would prefer having the lookup functions in AddressBook instead. For easier reuse later. What do you think?